Zulip /README.md

```diff
@@ -0,0 +1,42 @@
+# Hi :smiley: :wave:
+
+
+## If you’re new here, here’s what you can do to get started with this community:
+
+:heavy_check_mark: [Introduce yourself on #newcomers in the #newcomers > Introductions topic and tell us your motivations for joining so we can get to know you and help you on your journey](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers)
+
+:calendar: [Follow our Open Sessions in Calendar Events topic on #announcements or through our AnitaB.org Open Source calendar (PST:clock1:). These are voice calls you can participate in and get to know a bit more our community and projects](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles)
+
+:eyeglasses: Learn about our [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/code_of_conduct.md) and [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/reporting_guidelines.md) to keep this community safe for you and others.
+
+# [To get Acquainted with Zulip](https://zulipchat.com/help/getting-started-with-zulip)  
+
+# Here are the main streams you should be aware of:
+
+1. Ask your questions  on [questions :question:](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions) or [newcomers :new:](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers) streams 
+    
+2. Join streams of your interest such as 
+    1. [coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding):computer:
+    2. [design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design):art:
+    3. [documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentatio):notebook_with_decorative_cover:
+    4. [outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research):microscope:
+    5. [oppoturnites](https://anitab-org.zulipchat.com/#narrow/stream/223069-opportunities):stars:
+    6. [quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance):heavy_check_mark:
+
+3. You can join also project-specific streams, such as: 
+    1. [mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system):mortar_board:
```

Comment:
for each project you can include the repositories which are related to each stream. For example portal has [anitab-org/portal](https://github.com/anitab-org/portal) while mentorship system has 4 repos (mobiles and backend)